source/reference/logging.txt

```diff
@@ -43,7 +45,7 @@ Log to File
 ^^^^^^^^^^^
 
 To output log messages to file, start ``mongosync`` with the 
-:setting:`logPath` setting or :option:`--logPath` option.
+:option:`--logPath` option or use the :setting:`logPath` setting.
```

Comment:
I think it helps to say 'set the :setting:logPath in the configuration file' so that users know where to set the value.

Reply:
Fixed